### PR TITLE
Clarify unknown or null behavior

### LINKED ISSUE #20: Remove "useDefaults" from ImprovedBuilding schemas
This property is superseded by automatedMeasures.

### PR DESCRIPTION
**Pull request by kloconnell**

### CODE REVIEW

**Comment by kloconnell** [comment]
@joshuamzm how do I get my new file `defaulting_engine.rst` to show up in the index of pages?

**Comment by joshuamzm** [comment]
> @joshuamzm how do I get my new file `defaulting_engine.rst` to show up in the index of pages?

@kellyatradiantlabs I added the Sphinx link (without the first underscore) in the `toctree` section of the index file ([fbf6302](https://github.com/RadiantLabs/modeling-api-docs/pull/21/commits/fbf63025088d23e98fa41fd9020b4c73a36e0fd1)).

Now you can see your page linked in the ToC and in the sidebar.

image

I fixed the JSON code blocks formatting and couple typos in subsequent commits.

**Review by joshuamzm — APPROVED** [review]

File: docs/source/defaulting_engine.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.. _defaulting_engine:
+
+Defaulting Engine
+=================
+
+Radiant Labs understands that API users may not always know all of the details about the building being modeled. We have built a logical, statistics-based
+system to define default building characteristics. This defaulting engine uses tax assessor data, building permits, and NREL's ResStock building stock data.
+
```
Comment (jefffriesen) [inline docs/source/defaulting_engine.rst:8]: @kellyatradiantlabs naming ResStock is slightly too much information for my comfort level. Tax assess and permits is ok. Can you describe how ResStock got the data? Such as "regional building characteristics based on historical tax codes and x, y, z".

**Review by jefffriesen — CHANGES_REQUESTED** [review]

**Review by jefffriesen — COMMENTED** [review]
LGTM

**Pull request merged**